docs/manual/external-tools.tex

```diff
@@ -750,6 +753,21 @@
 \url{https://youtrack.jetbrains.com/issue/IDEA-159286}.
 
 
+\subsectionAndLabel{Running a checker on every IntelliJ change or save}{intellij-every-save}
+
+To make IntelliJ compile on every change or save,
+access Settings (Preferences on macOS).
+Select Build, Execution, Deployment > Compiler.
+Enable Build project automatically. Press OK.
+
+You can also configure IntelliJ to automatically save (and thus
+automatically compile) your work periodically.
+
+You may also wish to do the following:
+Press Ctrl+Shift+A (Cmd+Shift+A on macOS) and search for Registry.
```

Comment:
When I do this, the dialog box, in red letters, says "Changing these values may cause unwanted behavior of IntelliJ IDEA.  Please do not change these values unless you have been asked."  So, I don't think we should recommend this.